.env.example

```diff
@@ -23,10 +23,13 @@ E2E_PROOF_TYPE=groth16
 # Optional tag for eth light client to use in E2E tests
 # Either an empty string, or 'local', means it will use the local binary in the repo, unless running in mock mode
 # Otherwise, it will download the binary from the release at the specified tag
-E2E_WASM_LIGHT_CLIENT_TAG="cw-ics08-wasm-eth-vX.Y.Z"
+E2E_WASM_LIGHT_CLIENT_TAG="local"
```

Comment:
In this case, `cw-ics08-wasm-eth-vX.Y.Z` format should be mentioned in the comments above